common/djangoapps/student/views.py

```diff
@@ -599,17 +599,17 @@ def dashboard(request):
         settings.FEATURES.get('DISPLAY_COURSE_MODES_ON_DASHBOARD', True)
     )
 
-    # we want to filter and only show enrollments for courses within
-    # the 'ORG' defined in configuration.
-    course_org_filter = configuration_helpers.get_value('course_org_filter')
+    # We want to filter and only show enrollments for courses within
+    # the organizations defined in configuration for the current site.
+    course_org_filter = configuration_helpers.get_current_site_orgs()
 
     # Let's filter out any courses in an "org" that has been declared to be
     # in a configuration
     org_filter_out_set = configuration_helpers.get_all_orgs()
 
-    # remove our current org from the "filter out" list, if applicable
+    # Remove current site orgs from the "filter out" list, if applicable
```

Comment:
Yeah, let's move the `course_org_filter` stuff from above to here -- that will improve readability and does not appear to impact the overall flow